examples/angular/src/pages/ui/components/authenticator/use-authenticator/use-authenticator.component.html

```diff
@@ -0,0 +1,12 @@
+<amplify-authenticator initialState="signUp">
+  <ng-template amplifySlot="sign-up-footer">
+    <button amplify-button fontWeight="normal" size="small" variation="link">
+      Forgot your password?
+    </button>
+  </ng-template>
+</amplify-authenticator>
+
+<ng-container *ngIf="authenticator.route === 'authenticated'">
+  <h2>Welcome, {{ authenticator.user.username }}!</h2>
+  <button (click)="authenticator.signOut()">Sign Out</button>
+</ng-container>
```

Comment:
@ErikCH, can you help me with creating this example for Vue?

(and @ericclemmons once React slots are implemented!)

Reply:
![Screen Shot 2021-11-11 at 1 32 17 PM](https://user-images.githubusercontent.com/43682783/141372141-d551ca14-49e2-4f05-98b9-9a4ce4c28046.png)